docs/main.adoc

```diff
@@ -9,18 +9,22 @@ Direktoratet for forvaltning og IKT - Versjon 1.1
 :sectlinks:
 
 :leveloffset: +1
+:table-stripes: even
 
 Status: Gjeldende +
 Publisert: 11.10.2016 +
 Lenke til siste versjon: https://doc.difi.no/dcat-ap-no/ +
 
 Dette dokumentet beskriver “Standard for beskrivelse av datasett og datakataloger (DCAT-AP-NO)” som skal sikre at beskrivelser av offentlige data utføres på en felles, strukturert måte og i en maskinlesbar form. Standarden stiller krav til hva som _skal_, _bør_ og _kan_ være med i beskrivelsene, med spesifikasjon av dataformat.
 
-NOTE: *Innmelding av feil og mangler:* +
-Dersom du finner feil eller mangler i dokumentet, ber vi om at dette meldes inn på https://github.com/difi/dcat-ap-no/issues[Github Issues]. Dersom man ikke allerede har bruker på Github kan man opprette bruker gratis. +
+NOTE: *En ny versjon av denne standarden er under utvikling og vil bli lagt ut på høring i løpet av høsten 2020*
+Lenke til arbeidsdokument for versjon 2.0: (lenke kommer)
+
+*Innmelding av feil og mangler:* +
+Dersom du finner feil eller mangler i dokumentet, ber vi om at dette meldes inn på https://github.com/informasjonsforvaltning/dcat-ap-no/issues[Github Issues]. Dersom man ikke allerede har bruker på Github kan man opprette bruker gratis. +
  +
- *Test datakatalogen din med Difis valideringsverktøy:* +
- Difi har utviklet https://difi.github.io/dcat-ap-no-validator/[en online validator for testing av DCAT-AP-NO-datakataloger]
+ *Test datakatalogen din med Digitaliseringsdirektoratets valideringsverktøy:* +
+ Digitaliseringsdirektoratet har utviklet https://difi.github.io/dcat-ap-no-validator/[en online validator for testing av DCAT-AP-NO-datakataloger]
```

Comment:
Er det ikke bedre å flytte på validator først til informasjonsforvaltning, slik at URLen blir korrekt?

Reply:
Nei, for det er litt jobb. Vi endrer url til validator når validator er flytta.